app/components/moderation-list-row/template.hbs

```diff
@@ -1,25 +1,23 @@
-{{#link-to 'preprints.provider.preprint-detail' theme.id submission.id disabled=dataLoading}}
+{{#link-to 'preprints.provider.preprint-detail' theme.id submission.id
+           disabled=(or moderatorLoading submission.node.isPending submission.node.contributors.isPending)}}
```

Comment:
This is fine for now, but feels kinda messy. Before long we should start using ember-concurrency to encapsulate all this loading into a single task.